src/main/proto/tech/figure/util/v1beta1/credit_report.proto

```diff
@@ -53,4 +54,16 @@ message RiskModel {
     string name        = 1;
     string description = 2;
   }
+}
+
+/*
+Eligibility factors taken directly from the credit report
+ */
+message EligibilityFactors {
+  repeated Factor factors = 1;
+  message Factor {
+    string factor_type  = 1;
+    string msg          = 2;
```

Comment:
Nit: does this need to be `msg` in order to avoid conflicts or can we make it `message`?